authentication/tokens.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import requests
+from django.http.response import HttpResponseRedirect
+from rest_framework_simplejwt.tokens import RefreshToken
+
+from authentication.models import User
+
+
+def gen_tokens(email, password):
+    return requests.post(os.getenv("DOMAIN") + "auth/token/", data={"email": email, "password": password}).json()
```

Comment:
Do we really need to make a request from within our app to another end-point of our app? Let's try to make the initial request directly to this URL instead of making requests internally.